### PR TITLE
Extended event notification section

### PR DESCRIPTION
**Pull request by AndreaFilyo**
Extended event notification section with more details.

### CODE REVIEW

File: inventory-api/concepts/how-inventories-work.md [hunk 1]
```diff
@@ -82,9 +82,22 @@ The request body contains the parameters for the inventory flow. In the case of
...
+You can for example create a subscription for the `InventoryBalanceChanged` event. Whenever the inventory gets updated for example through a purchase, the `InventoryBalanceChanged` event occurs. Your application will then receive event data on the destination server. The event data is the payload for the updated inventory. 
+
+See [Pusher API](https://github.com/iZettle/api-documentation/blob/master/pusher-api/overview.md) for more details.
+
+The following events are available for the inventory:
+
+* `InventoryBalanceChanged`.  The product inventory was updated from Zettle POS or the app.
+* `InventoryTrackingStarted`. Tracking of items in the product library was initiated from Zettle POS or the app.
+* `InventoryTrackingStopped`. Tracking of items in the product library was disabled from Zettle POS or the app.
+
```
Comment (Astro-Choco) [inline inventory-api/concepts/how-inventories-work.md:100]: I think this info can be moved after the first paragraph at row 88.

File: inventory-api/concepts/how-inventories-work.md [hunk 2]
```diff
@@ -82,9 +82,22 @@ The request body contains the parameters for the inventory flow. In the case of
...
 See [Update inventory balance](../user-guides/manage-inventory-balances/update-inventory-balance.md). 
 
-### Event notifications
+### Inventory events
+
+Using the Zettle Pusher API, your application can get notified about changes to the inventory. You can create a webhook subscribing to inventory-related events. When the event occurs, a notification is sent to the specified notification URL. The notification is a `POST` request with  event details in JSON format.
+
 Inventory balance updates can be the result of either a purchase event, or an explicit update request. Such a request will trigger webhook event notifications for balance updates similar to purchase events.
 
+You can for example create a subscription for the `InventoryBalanceChanged` event. Whenever the inventory gets updated for example through a purchase, the `InventoryBalanceChanged` event occurs. Your application will then receive event data on the destination server. The event data is the payload for the updated inventory.
```
Comment (Astro-Choco) [inline inventory-api/concepts/how-inventories-work.md:91]: For example, your app subscribes to the `InventoryBalanceChanged` event. When a purchase happens, your app will receive data of the `InventoryBalanceChanged` event on the destination server, The event data is the payload for the updated inventory.

**Review by Astro-Choco — COMMENTED** [review]
Looks good except some minor feedback

File: inventory-api/concepts/how-inventories-work.md (hunk 1 above)
Comment (AndreaFilyo) [reply inventory-api/concepts/how-inventories-work.md:100]: Fixed

File: inventory-api/concepts/how-inventories-work.md (hunk 2 above)
Comment (AndreaFilyo) [reply inventory-api/concepts/how-inventories-work.md:91]: Fixed

**Review by Astro-Choco — APPROVED** [review]
Good to have this info

**Pull request merged**